ansible/roles/build-device-worker/defaults/main.yaml

```diff
@@ -1,4 +1,4 @@
 ---
-base_repo_dir: /tmp/build
+base_repo_dir: /workspace/workspace/
 rpm_build_directory: ~/rpmbuild
 build_arch: x86_64
```

Comment:
You're not using build_arch in this role (since you're building both targets)

Reply:
I'll remove it.